README.md

```diff
@@ -27,20 +24,40 @@
- |
-  <a aria-label="expo documentation" href="https://docs.expo.dev">Read the Documentation 📚</a>
+&ensp;•&ensp;
+  <a aria-label="expo documentation" href="https://docs.expo.dev">Read the Documentation</a>
+&ensp;•&ensp;
+  <a aria-label="expo documentation" href="https://blog.expo.dev">Learn more on our blog</a>
+&ensp;•&ensp;
+  <a aria-label="expo documentation" href="https://expo.canny.io/feature-requests">Request a feature</a>
 </p>
 
-<p>
-  <a aria-label="Follow @expo on Twitter" href="https://twitter.com/intent/follow?screen_name=expo" target="_blank">
-    <img  alt="Twitter: expo" src="https://img.shields.io/twitter/follow/expo.svg?style=flat-square&label=Follow%20%40expo&logo=TWITTER&logoColor=FFFFFF&labelColor=00aced&logoWidth=15&color=lightgray" target="_blank" />
-  </a>
-  <a aria-label="Follow Expo on Medium" href="https://blog.expo.dev">
-    <img align="right" alt="Medium: exposition" src="https://img.shields.io/badge/Learn%20more%20on%20our%20blog-lightgray.svg?style=flat-square" target="_blank" />
+<h6 align="center">Follow us on</h6>
+<p align="center">
+  <a aria-label="Follow @expo on X" href="https://twitter.com/intent/follow?screen_name=expo" target="_blank">
+    <img alt="Expo on X" src="https://img.shields.io/badge/X-000000?style=for-the-badge&logo=x&logoColor=white" target="_blank" />
+  </a>&nbsp;
+  <a aria-label="Follow @expo on GitHub" href="https://github.com/expo" target="_blank">
+    <img alt="Expo on GitHub" src="https://img.shields.io/badge/GitHub-222222?style=for-the-badge&logo=github&logoColor=white" target="_blank" />
+  </a>&nbsp;
+  <a aria-label="Follow @expo on Reddit" href="https://www.reddit.com/r/expo/" target="_blank">
+    <img alt="Expo on Reddit" src="https://img.shields.io/badge/Reddit-FF4500?style=for-the-badge&logo=reddit&logoColor=white" target="_blank" />
+  </a>&nbsp;
+  <a aria-label="Follow @expo on Bluesky" href="https://bsky.app/profile/expo.dev" target="_blank">
+    <img alt="Expo on LinkedIn" src="https://img.shields.io/badge/Bluesky-1DA1F2?style=for-the-badge&logo=bluesky&logoColor=white" target="_blank" />
+  </a>&nbsp;
+  <a aria-label="Follow @expo on LinkedIn" href="https://www.linkedin.com/company/expo-dev" target="_blank">
+    <img alt="Expo on LinkedIn" src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white" target="_blank" />
   </a>
 </p>
-  
----
+
+## Introduction
+
+Expo is an open-source platform for making universal native apps that run on Android, iOS, and the web. It includes a universal runtime and libraries that let you build native apps by writing React and JavaScript. This repository is where the Expo client software is developed, and includes the client apps, modules, apps, CLI, and more. [Expo Application Services (EAS)](https://expo.dev/eas) is a platform of hosted services that are deeply integrated with Expo open source tools. EAS helps you build, ship, and iterate on your app as an individual or a team.
```

Comment:
i'd change this "This repository" sentence to:

> This repository includes the Expo SDK, Modules API, Go app, CLI, Router, documentation, and various other supporting tools.

Reply:
ah, i saw this wasn't even written in this pr, so i made the change myself here https://github.com/expo/expo/commit/58250e142307873fca65eb2fa76229fcd17187d4